lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
Thanks. I'd prefer to wait for the feedback of Bertrand, our local Windows expert. We expect him back on Thursday. In principle I'd prefer to have the dynamic library handling fixed, rather than sneaking this dlsym call in here. The llvm::DynLibraries are so unhelpful that we have considered getting rid of them. Maybe your suggestion is the right way.

Do you have timing info? How long does it take to enumerate all libs? How long does it take to find a symbol per library? We have users with hundreds of shared libs; I'm a bit scared of that loop...